draft-ietf-tls-tls13.md

```diff
@@ -4037,6 +4046,28 @@ lacking a "server_name" extension by terminating the connection with a
 Security issues are discussed throughout this memo, especially in
 {{implementation-notes}}, {{backward-compatibility}}, and {{security-analysis}}.
 
+## Re-Using Ephemeral Shares
+
+This document refers to client and server key shares as ephemeral. This literally
+means that they are used for a short time. Keeping the private key secret forever
+is necessary for the property of Perfect Forward Secrecy. The simplest and safest 
+way to achieve this is to generate a new Diffie-Hellman share for each TLS 
+handshake, and to destroy keys immediately after they are used. This has some 
+cost in processing power. 
+
```

Comment:
@prosecco @tvdmerwe thanks.

I agree it's clear that you don't get PFS if you don't use fresh DH keys. I think the question is whether the AKE properties of your proofs depend on the DH keys being fresh. @tvdmerwe is this something you could relax in the Tamarin model easily?

Basically, my concern here is that this text seems to say that it's OK to use non-fresh DH shares for 1.3, with the cost being PFS. If we're to incorporate text like that, it would be nice to have some analytic results which indicate that it's true.